API/chat/tests.py

```diff
@@ -412,7 +424,8 @@ def patch_and_get_response(self, messageid, text, timestamp, typing):
         })
         return self.client.patch(
             reverse('chat:message', args=(messageid,)),
-            qstring
+            qstring,
+            **{'HTTP_AUTHORIZATION': settings.PASS}
```

Comment:
We could have this header by default on the `patch` request because we've created our own client class for this kind of request, but what could we do on the POST request? We're just adding headers.